.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Issue tracker is **ONLY** used for reporting bugs.
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Context (Environment)
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Detailed Description
+<!--- Provide a detailed description of the change or addition you are proposing -->
+
+## Possible Implementation
+<!--- Not obligatory, but suggest an idea for implementing addition or change -->
```

Comment:
Is that meaningful in bug reports? I mean, one could do proposal how to fix but this is not required in general. Also, the text should be "<!--- Provide a detailed description of the fix you are proposing -->" and "<!--- Not obligatory, but suggest an idea for implementing the fix -->"

Reply:
Copied from `vehicle-app-python-sdk`. Let's keep the files in sync.